### PR TITLE
Adds new log interface for logging to stdout and replaces all fmt prints with the new interface

### PR DESCRIPTION
**Pull request by aqt01**
Closes:
- #391 
- #396 
- #402 
- https://github.com/pokt-network/pocket-core/issues/412

### CODE REVIEW

File: logs/models.go [hunk 1]
```diff
@@ -13,7 +12,29 @@ var (
...
-	JSONLogFormat = LogFormat{&logmatic.JSONFormatter{}}
+	JSONLogFormat = LogFormat{&logrus.JSONFormatter{
+			DisableTimestamp:			false,
+			TimestampFormat:			"2006-01-02 15:04:05",
+			DataKey:					"",
+			FieldMap:					nil,
+			CallerPrettyfier:			nil,
+			PrettyPrint:				false,
+	}}
+	TextLogFormatter = LogFormat{&logrus.TextFormatter{
```
Comment (luyzdeleon) [inline logs/models.go:23]: Let's create an issue to create a config to edit all these values via configuration file or flags.

**Review by luyzdeleon — CHANGES_REQUESTED** [review]
Can we rename the `NewLog` function to something more expressive? Also we need comments on all functions and a unit test.

File: logs/models.go (hunk 1 above)
Comment (andrewnguyen22) [reply logs/models.go:23]: Make sure you use `gofmt` before pushing to format to spec

**Review by andrewnguyen22 — CHANGES_REQUESTED** [review]

File: logs/models.go (hunk 1 above)
Comment (aqt01) [reply logs/models.go:23]: @andrewnguyen22  
https://github.com/pokt-network/pocket-core/pull/441/commits/349a75aa0c292f76903432011bd873af328cfa51

@luyzdeleon 
https://github.com/pokt-network/pocket-core/pull/441/commits/c99e86fbb9d9403ec280be779c2d090a9ca688e5

**Review by andrewnguyen22 — APPROVED** [review]

**Review by luyzdeleon — APPROVED** [review]

**Pull request merged**